common/djangoapps/course_modes/tests/test_views.py

```diff
@@ -397,6 +400,32 @@ def test_verified_mode_creation(self, mode_slug, mode_display_name, min_price, s
 
         self.assertEquals(course_mode, expected_mode)
 
+    @override_waffle_flag(DISCOUNT_APPLICABILITY_FLAG, active=True)
+    @override_waffle_flag(FIRST_PURCHASE_OFFER_BANNER_DISPLAY, active=True)
+    def test_discount_on_track_selection(self):
```

Comment:
sure i can use patch instead if that's what you prefer.  
although typically what we do in unit tests like the ones for FBE is create the necessary configs, i think we usually add a verified mode for a course rather than patching it for example

Reply:
changed